credentials/model_entity_array__embedded_items_inner.go

```diff
@@ -0,0 +1,105 @@
+/*
+PingOne Platform API - Credentials
+
+The PingOne Platform API covering the PingOne Credentials service
+
+API version: 2023-03-30
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package credentials
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// EntityArrayEmbeddedItemsInner struct for EntityArrayEmbeddedItemsInner
+type EntityArrayEmbeddedItemsInner struct {
+	CredentialType *CredentialType
+	UserCredential *UserCredential
+}
+
+// Unmarshal JSON data into any of the pointers in the struct
+func (dst *EntityArrayEmbeddedItemsInner) UnmarshalJSON(data []byte) error {
+	var err error
+	// try to unmarshal JSON data into CredentialType
+	err = json.Unmarshal(data, &dst.CredentialType);
+	if err == nil {
+		jsonCredentialType, _ := json.Marshal(dst.CredentialType)
+		if string(jsonCredentialType) == "{}" { // empty struct
```

Comment:
This needs postprocessing, because there may be a possibility that the `Unmarshal`+`Marshal` routine matches both `dst.*` objects and returns the wrong one.  An example of the additional data "type" check I do in the `else` of this `if` block is here: https://github.com/patrickcping/pingone-go-sdk-v2/blob/eeeef1392b02f9fa529ab907856f2ace83b6fc29/management/model_entity_array__embedded_gateways_inner.go#L51-L55

Reply:
Because of the post processing change, it'll need adding to the postprocessor script:
https://github.com/patrickcping/pingone-go-sdk-v2/blob/main/scripts/generate-replace-regex.go

Reply:
There isn't as clean of a way, but I believe the following is sufficient.

For CredentialType, I check if .CardDesignTemplate is not null.  Only CredentialType will have a design template.
```
if dst.CredentialType.CardDesignTemplate != "" {
	return nil // data stored in dst.CredentialType, return on the first match
} else {
	dst.CredentialType = nil
}
```  

For UserCredential, I check if the .User.Id is not null, which will only be present on the User Credential object.
```
if dst.UserCredential.User.Id != "" {
	return nil // data stored in dst.UserCredential, return on the first match
} else {
	dst.UserCredential = nil
}
```

Reply:
> Because of the post processing change, it'll need adding to the postprocessor script: https://github.com/patrickcping/pingone-go-sdk-v2/blob/main/scripts/generate-replace-regex.go

I need to sync up with you on this.  I'm not 100% clear what to update here.  I've modified the model_entity_array... but need a little help with the postprocessor script update.

Reply:
Resolved. Problem was how I invoked the generate script. Instead of proper `make generate` which passes in the appropriate github user and repo name, I was invoking the generate script directly, which is the cause of the perceived postprocessor script issue.  Issue closed.